tests/non_matching/mapping_info.cc

```diff
@@ -464,6 +464,87 @@ test(const bool filtered_compression)
           << dst_faces.l2_norm() - dst_faces_2.l2_norm() << std::endl;
 }
 
+
+
+void
+test_vec_of_qpoints()
```

Comment:
This should be its own test `mapping_info_02` in my opinion.

Reply:
Yes, I agree, simply move this to a new file `mapping_info_02` and create that as a new test. Thanks for adding, this is highly appreciated.

Reply:
So done, thank you!